src/main.c

```diff
@@ -676,6 +683,7 @@ int LCUI_Destroy(void)
 	e.type = LCUI_QUIT;
 	LCUI_TriggerEvent(&e, NULL);
 	System.state = STATE_KILLED;
+	LCUI_UnbindEvent(LCUI_SETTINGS_CHANGE);
```

Comment:
The expected parameter of `LCUI_UnbindEvent()` is handler_id instead of event_id, you should store the return value of `LCUI_BindEvent()` and pass it to `LCUI_UnbindEvent()`.

https://github.com/lc-soft/LCUI/blob/879886f31347e4817c4e94d3e6e8bdc3c8d5bd0d/src/main.c#L275-L285